packages/editor/playground/playground.tsx

```diff
@@ -57,9 +57,17 @@ const BlurHelper: FC = () => {
 
 const DebugConsole: FC<{ hasUnsavedChanges: boolean; content: string }> = ({ hasUnsavedChanges, content }) => {
     return (
-        <div>
-            <p>content: {content}</p>
-            <p>hasUnsavedChanges: {JSON.stringify(hasUnsavedChanges)}</p>
+        <div className="css-t4vd4r" style={{ borderTop: "1px solid gray" }}>
```

Comment:
Where does the className `css-t4vd4r` come from? I don't think we need a className here.